### PR TITLE
PT-156609771: Test synchronizing far ahead node

### PR DESCRIPTION
**Pull request by eproxus**
A new performance suite is added, and the existing sync speed test is moved there. A new startup speed test is added.

Faster `smoke-test` target will be added in a separate PR.

### CODE REVIEW

**Comment by sylane** [comment]
Maybe some separate testing for long-running performance tests ? `make perf-tests` ?

File: system_test/helpers/aest_docker_api.erl [hunk 1]
```diff
@@ -207,6 +208,11 @@ container_logs(ID) ->
             throw({docker_error, maps:get(message, decode_json(Response))})
     end.
 
+commit(ID, Tag) ->
```
Comment (ThomasArts) [inline system_test/helpers/aest_docker_api.erl]: Probably a comment what commit is used for? Of course, I could read the manual of dockers.

**Comment by ThomasArts** [comment]
We have 8 hours for all system tests together (nightly run). If we smartly re-use this docker container, we could benefit from creating a long chain once.

File: system_test/helpers/aest_docker_api.erl (hunk 1 above)
Comment (eproxus) [reply system_test/helpers/aest_docker_api.erl]: Commit is the standard command in Docker to create an image from a container file system

**Comment by eproxus** [comment]
@sylane No, it's coming in a separate PR

**Comment by eproxus** [comment]
See the description

**Comment by sylane** [comment]
I should really be more attentive...

**Pull request merged**